### PR TITLE
feat(onboarding): remove page types introduction screen

### PR DESCRIPTION
**Pull request by jomifepe**
**Resolves**: DT-2231

### Description

Removing the following screen:

Screenshot 2024-07-25 at 09 51 22

And go straight to:

Screenshot 2024-07-25 at 09 52 48

### Checklist

- [ ] If my changes require tests, I added them.
- [ ] If my changes affect backward compatibility, it has been discussed.
- [ ] If my changes require an update to the CONTRIBUTING.md guide, I updated it.

### How to QA [^1]

[^1]:
	Please use these labels when submitting a review:
	:question: #ask:&ensp;Ask a question.
	:bulb: #idea:&ensp;Suggest an idea.
	:warning: #issue:&ensp;Strongly suggest a change.
	:tada: #nice:&ensp;Share a compliment.

### CODE REVIEW

**Review by dani-mp — APPROVED** [review]

File: packages/slice-machine/src/pages/index.tsx [hunk 1]
```diff
@@ -1,16 +1,8 @@
 import { type FC } from "react";
 
 import { CustomTypesTablePage as CustomTypesTablePageTemplate } from "@/features/customTypes/customTypesTable/CustomTypesTablePage";
-import { PageTypeOnboarding } from "@/features/customTypes/PageTypeOnboarding";
-import { useIsEmptyProject } from "@/hooks/useIsEmptyProject";
```
Comment (dani-mp) [inline packages/slice-machine/src/pages/index.tsx:5]: just to be sure, this one is still being used elsewhere? (to remove it otherwise)

Comment (jomifepe) [reply packages/slice-machine/src/pages/index.tsx:5]: No, actually not being used anywhere else, looked like a useful hook to leave 😄

Comment (jomifepe) [reply packages/slice-machine/src/pages/index.tsx:5]: ~Removed~
Correction: It's still being used by the existing onboarding guide, which I will remove next.
I started to remove it locally already, so I didn't see any other usages.

**Review by BohdanOne — APPROVED** [review]
Looks good! 
Of course if removed components aren't used elsewhere, what Dani suggested to double-check.

File: packages/slice-machine/src/pages/index.tsx (hunk 1 above)
Comment (jomifepe) [reply packages/slice-machine/src/pages/index.tsx:5]: Correction: It's still being used by the existing onboarding guide, which I will remove next.
I started to remove it already, so I didn't see any other usages.

**Pull request merged**